website/content/en/preview/getting-started/scripts/step12-add-provisioner.sh

```diff
@@ -0,0 +1,20 @@
+cat <<EOF | kubectl apply -f -
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:
+    - key: karpenter.sh/capacity-type
+      operator: In
+      values: ["spot"]
+  limits:
+    resources:
+      cpu: 1000
+  provider:
+    subnetSelector:
+      karpenter.sh/discovery: ${CLUSTER_NAME}
+    securityGroupSelector:
+      karpenter.sh/discovery: ${CLUSTER_NAME}
+  ttlSecondsAfterEmpty: 30
```

Comment:
Consider adding the Expired feature as well.
```suggestion
  ttlSecondsAfterEmpty: 30
  ttlSecondsAfterExpired: 86400
```

Reply:
Not sure what you mean, but I'm not planning to add anything new to the scripts, they are moved as is.